python/instrumentation/openinference-instrumentation-openai/src/openinference/instrumentation/openai/_request.py

```diff
@@ -208,6 +211,17 @@ def _finalize_response(
                     response_accumulator=response_accumulator,
                 )
                 return response
+            if (
```

Comment:
Can you link the OpenAI code here so we can grok this later? Hard to understand the Boolean logic without

Reply:
added